File: app/src/main/java/org/neidhardt/dynamicsoundboard/views/edittext/NoUnderscoreEditText.java
package org.neidhardt.dynamicsoundboard.views.edittext;

import android.content.Context;
import android.util.AttributeSet;
import org.neidhardt.dynamicsoundboard.R;


public class NoUnderscoreEditText extends CustomEditText
{
	public NoUnderscoreEditText(Context context, AttributeSet attrs)
	{
		super(context, attrs);
	}

	@Override
	protected int getLayoutId()
	{
		return R.layout.view_dialog_edittext;
	}
}

File: app/src/main/kotlin/org/neidhardt/dynamicsoundboard/views/edittext/NoUnderscoreEditText.kt
package org.neidhardt.dynamicsoundboard.views.edittext

import android.content.Context
import android.util.AttributeSet
import android.view.LayoutInflater
import kotlinx.android.synthetic.main.view_no_underscore_edittext.view.*
import org.neidhardt.dynamicsoundboard.R
import org.neidhardt.ui_utils.views.CustomEditText
import org.neidhardt.ui_utils.views.EditTextBackEvent


class NoUnderscoreEditText(context: Context, attrs: AttributeSet) : CustomEditText(context, attrs) {

	private var inputField: EditTextBackEvent? = null
	override var input: EditTextBackEvent
		get() = inputField as EditTextBackEvent
		set(value) { this.inputField = value }

	override var mOnTextEditedListener: OnTextEditedListener? = null

	override fun inflateLayout(context: Context) {
		LayoutInflater.from(context).inflate(R.layout.view_no_underscore_edittext, this, true)
		this.input = this.et_view_no_underscore_edittext
	}
}
